.gitignore

```diff
@@ -1,23 +1,335 @@
-# HOWTO with the infos how the git was created
-HOWTO.sh
-# all temporary emacs and vim files
+# Created by https://www.gitignore.io/api/vim,code,emacs,python,pycharm,virtualenv,sublimetext
+# Edit at https://www.gitignore.io/?templates=vim,code,emacs,python,pycharm,virtualenv,sublimetext
+
+### Code ###
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+
+### Emacs ###
+# -*- mode: gitignore; -*-
 *~
-# bam files
-PAQR/data/bam_files
-# data from PAQR test runs
-PAQR/log_output.log
-PAQR/HNRNPC_KD/
-PAQR/data/samplewise_TIN/
-PAQR/.snakemake/
-PAQR/scripts/deNovo-used-sites-and-usage-inference.testVersion.py
-KAPAC/doIt.sh
-KAPAC/RESULTS/
-KAPAC/RESULTS_FORMER.log
-KAPAC/RESULTS_FORMER/
-KAPAC/RESULTS_FORMER_AFTER_CLEANUP/
-KAPAC/RESULTS.log
-# Rhistory
-*.Rhistory
-KAPAC/DATA/kmer_counts_all.tsv
```

Comment:
These are definitely leftovers, should be removed.

Reply:
yeah... red color means they have been removed, right?

Reply:
Oh, sorry, that was stupid of me....